pkg/rbac/userData.go

```diff
@@ -128,6 +129,7 @@ func (user *UserDataCache) getClusterScopedResources(cache *Cache, ctx context.C
 				Resource{Apigroup: res.Apigroup, Kind: res.Kind})
 		}
 	}
+	klog.V(7).Info("User has access to these cluster scoped res: ", user.userData.CsResources)
```

Comment:
Include the user id in case we need to debug multiple users.

Reply:
Added